types.d.ts

```diff
@@ -22,7 +22,12 @@ export interface BlogSettings {
   cover?: string;
   coverTextColor?: string;
   author?: string;
-  links?: { title: string; url: string; icon?: VNode }[];
+  links?: {
+    title: string;
+    url: string;
+    icon?: VNode;
+    target: "_self" | "_blank" | "_parent" | "_top";
```

Comment:
default to `_self` ?

Reply:
@ije 
Yes. We will default to "_self" to avoid user influence.
However, I have confirmed that it is not working correctly due to a flaw in the implementation, so create a fix commit.